zcash_client_backend/examples/diversify-address.rs

```diff
@@ -53,6 +53,8 @@ fn main() {
         return;
     };
 
+    let diversifier_index = opts.diversifier_index.unwrap_or_else(|| DiversifierIndex::new());
```

Comment:
This change does nothing, as `opts.diversifier_index` is used on the next line, and `diversifier_index` is then shadowed.